src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -61,6 +60,22 @@
   </Target>
 
   <Target
+    Name="_RunForCoreWithRID"
+    DependsOnTargets="_ResolveInputArguments;_CreateResponseFileForMvcRazorPrecompile"
+    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' AND '$(RuntimeIdentifier)'!=''">
+
+    <ItemGroup>
+      <_RazorCompilationProject Include="$(MSBuildProjectFullPath)">
+        <AdditionalProperties>RuntimeIdentifier=;MvcRazorOutputPath=$(MvcRazorOutputPath)</AdditionalProperties>
```

Comment:
Why is runtime identifier empty? `RuntimeIdentifier=;`

Reply:
We're basically trying to publish without a RID. It's passing in a `null` \ empty value.

Reply:
Ah. It's confusing because the name of the target is `_RunForCoreWithRID` 😄